src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Fruit> fruits = new HashMap<>();
```

Comment:
According to the encapsulation principle, the access to the `fruits` map should be restricted. It is currently package-private, which is not as restrictive as it could be. Consider making it private to ensure that the map is only accessible through the methods provided in the `Storage` class.